tests/rules/test_cat_dir.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from thefuck.rules.cat_dir import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command('cat foo', 'cat: foo: Is a directory'),
+    Command('cat /foo/bar/', 'cat: /foo/bar/: Is a directory'),
```

Comment:
The actual output contains a LINEFEED (`\n`) at the end, so should the test output: `'cat: foo: Is a directory\n'` and `'cat: /foo/bar/: Is a directory\n'`